index.js

```diff
@@ -84,40 +123,123 @@ module.exports = {
         'eslint-plugin-ember@latest',
         'eslint-plugin-n@latest',
         '@babel/eslint-parser@latest',
+        'ember-template-lint@latest',
+
+        '@ember/string@latest',
         'ember-resolver@latest',
         'ember-load-initializers@latest',
+        'qunit@latest',
+        'qunit-dom@latest',
+        'stylelint@latest',
+        'stylelint-prettier@latest',
+        'stylelint-config-standard@latest',
+        'concurrently@latest',
+
         // Needed for eslint
         'globals',
         'babel-plugin-ember-template-compilation',
         'prettier-plugin-ember-template-tag',
+
+        // Dependencies out of date from upstream
+        // (or waiting for the release train to catch up)
+        'ember-route-template',
```

Comment:
this is not true https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/package.json doesn't have this. It's not part of the release train, there is a RFC adding this.

I can concede that in the TypeScript blueprint writing a GTS application template is a lot better so I'm happy to include this but only when `--typescript` is used

Reply:
the comment block was meant for another group of deps it turns out -- I probably messed it up in a rebase :(

anywho, e-route-template is now in the TS block with a link to the RFC